src/lib/EventPhase.ts

```diff
@@ -1,8 +1,7 @@
 /**
  * An enum for possible event phases that an event can be in
  */
-const enum EventPhase
-{
```

Comment:
You could leave out the ENUM value assignment.

Reply:
I'll change it to a typescript enum

Reply:
Did you guys consider `const` might be added for a specific reason?

https://stackoverflow.com/questions/28818849/how-do-the-different-enum-variants-work-in-typescript

Reply:
Didn't know this difference! But what I meant with my previous comment you could use the computed assignment instead of assigning it yourself.
```
const enum Event {
    EVENT_ONE,
    EVENT_TWO,
}
```

Reply:
Ah didn't even know that! Why output larger code if you don't need the lookup object for the event phase, I'll revert this commit!